lib/ex_admin/show.ex

```diff
@@ -308,21 +298,21 @@ defmodule ExAdmin.Show do
           end
           markup_contents do
             association_filler(post, resource_key: "post_id", assoc_name: "tags",
-              assoc_key: "tag_id", assoc_model: MyApp.Tag, autocomplete: false)
+              assoc_key: "tag_id", autocomplete: false)
           end
         end
       end
   """
   defmacro association_filler(resource, opts) do
     quote bind_quoted: [resource: resource, opts: opts] do
-      required_opts = [:resource_key, :assoc_name, :assoc_key, :assoc_model]
+      required_opts = [:resource_key, :assoc_name, :assoc_key]
       unless MapSet.subset?(MapSet.new(required_opts), MapSet.new(Dict.keys(opts))) do
```

Comment:
Dict has been deprecated. I wonder if we should use `Map.keys` or `Keyword.keys` here (can't remember if opts is a map or a keyword list here...)